dev.Dockerfile

```
@@ -1,14 +1,20 @@
 FROM python:3.9
 
-COPY requirements.txt ./
+ENV PYTHONPATH /app:$PYTHONPATH
+ENV PYTHONUNBUFFERED 1
+ENV PATH="${PATH}:/root/.poetry/bin"
+
 RUN apt-get update \
-    && apt-get install python3-dev binutils libproj-dev gdal-bin -y \
-    && pip install --upgrade pip \
-    && pip install -r requirements.txt
+    && apt-get install python3-dev binutils libproj-dev gdal-bin -y
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+RUN poetry config virtualenvs.create false
+
+WORKDIR /app
+
+COPY poetry.lock pyproject.toml ./
+RUN poetry install --no-dev --no-root
```

Itt nem inkább így kellene?

```suggestion
RUN poetry install --no-root
```

---

Itt ugyebar azt jelenti a `--no-dev`, hogy a dev dependencyket nem kell telepiteni. A `dev.Dockerfile` akkor van hasznalatban, amikor a lokalisan futtatjuk az appot az adatbazissal egy docker compose stackben. A stacken belul nincs szuksegunk a dev dependencykre, a lintelest nem a stacken belul futtatjuk, hanem a host gepen.

---

Tudom, hogy azt jelenti, azért írtam (én nem a host gépen futtattam ezeket korábban sem, de akkor max most majd átállok erre)